pkg/virt-handler/options.go

```diff
@@ -54,11 +54,15 @@ func virtualMachineOptions(
 	}
 
 	if clusterConfig != nil {
+		boshDisplay := true
+		if clusterConfig.VGADisplayForEFIGuestsEnabled() {
+			boshDisplay = false
+		}
 		options.ExpandDisksEnabled = clusterConfig.ExpandDisksEnabled()
 		options.ClusterConfig = &cmdv1.ClusterConfig{
 			ExpandDisksEnabled:        clusterConfig.ExpandDisksEnabled(),
 			FreePageReportingDisabled: clusterConfig.IsFreePageReportingDisabled(),
-			BochsDisplayForEFIGuests:  clusterConfig.BochsDisplayForEFIGuestsEnabled(),
+			BochsDisplayForEFIGuests:  boshDisplay,
```

Comment:
Is this to highlight that bochs is the default value?
otherwise we can simply use `!clusterConfig.VGADisplayForEFIGuestsEnabled()`

Reply:
Yes, I meant to make it more self-explanatory but maybe `!clusterConfig.VGADisplayForEFIGuestsEnabled()` is good enough, let me know.

Reply:
I would prefer `!clusterConfig.VGADisplayForEFIGuestsEnabled()` just to stay aligned consistent with other settings. But if you want to leave it as is, it's ok for me